src/profile_service.rs

```diff
@@ -1,46 +0,0 @@
-use crate::{
-    proto::WebSocketRequestMessage,
-    push_service::{ServiceError, SignalServiceProfile},
-    websocket::SignalWebSocket,
-    ServiceAddress,
-};
-
-pub struct ProfileService {
-    ws: SignalWebSocket,
-}
-
-impl ProfileService {
-    pub fn from_socket(ws: SignalWebSocket) -> Self {
-        ProfileService { ws }
-    }
-
-    pub async fn retrieve_profile_by_id(
-        &mut self,
-        address: ServiceAddress,
-        profile_key: Option<zkgroup::profiles::ProfileKey>,
-    ) -> Result<SignalServiceProfile, ServiceError> {
```

Comment:
We're using this in WF; did it really just go away?

Reply:
No, it's in `PushService`, and the impl was duplicated. Signal-Android really doesn't use the websocket for service API calls, so I think we should stick with it (especially since soon ™️  it will be using the same client behind the scenes).